### PR TITLE
Parameterize settings for SFTP usage

### PR DESCRIPTION
**Pull request by Phil-Friderici**
This fully backward compatible patch parameterize the settings to be used for a SFTP server setup:
ChrootDirectory
ForceCommand 
Match

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -452,6 +455,20 @@
     }
   }
 
+  if $sshd_config_chrootdirectory != undef {
+    validate_absolute_path($sshd_config_chrootdirectory)
+  }
+
+  if $sshd_config_forcecommand != undef {
+    if is_string($sshd_config_forcecommand) == false {
```
Comment (ghoneycutt) [inline manifests/init.pp]: validate_string() would have the same functionality without the extra logic

File: Gemfile [hunk 2]
```diff
@@ -5,3 +5,5 @@ gem 'puppet', puppetversion
 gem 'puppetlabs_spec_helper', '>= 0.1.0'
 gem 'puppet-lint', '>= 1.0.0'
 gem 'facter', '>= 1.7.0'
+gem 'rspec-puppet', '~>1.0'
```
Comment (ghoneycutt) [inline Gemfile]: Is this actually needed?

File: Gemfile [hunk 3]
```diff
@@ -5,3 +5,5 @@ gem 'puppet', puppetversion
...
+gem 'rspec', '~>2.0'
```
Comment (ghoneycutt) [inline Gemfile]: For consistency between modules, could you change the rspec line to match this

https://github.com/ghoneycutt/puppet-module-pam/blob/master/Gemfile#L14-17

**Comment by ghoneycutt** [comment]
Hi @Phil-Friderici 

I went ahead and made the changes in PR #106

File: manifests/init.pp (hunk 1 above)
Comment (Phil-Friderici) [inline manifests/init.pp]: validate_string() accepts numbers as string, is_string() does not. Therefore the extra mile here.

**Pull request closed without merging**